<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ECG Carpets</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="CarpetView" name="carpetView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="PlotWidget" name="signalView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="openPushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Open file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="fixedHeightCheckBox">
           <property name="text">
            <string>fixed height</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="fixedHeightSpinBox">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000000</number>
           </property>
           <property name="singleStep">
            <number>50</number>
           </property>
           <property name="value">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Lead</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="leadComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>R source</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="rSourceLeadComboBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="autolevelsPushButton">
           <property name="toolTip">
            <string>Cut-off top and bottom percentile</string>
           </property>
           <property name="text">
            <string>Auto-levels</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="autolevelsSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>x scale units</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QRadioButton" name="msRadioButton">
           <property name="text">
            <string>ms</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QRadioButton" name="bpmRadioButton">
           <property name="text">
            <string>bpm</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="10" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Color map</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QComboBox" name="cmapComboBox">
           <item>
            <property name="text">
             <string>jet</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>turbo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>seismic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bwr</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gray</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gray_r</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Font size</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QSpinBox" name="fontSizeSpinBox">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>36</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Theme</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QComboBox" name="themeComboBox">
           <item>
            <property name="text">
             <string>dark</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>light</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Line width</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QSpinBox" name="lineWidthSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>left offset</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>right offset</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="leftSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>5.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="rightSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>5.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QSpinBox" name="xTicksSpinBox">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>num x ticks</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="exportImagePushButton">
         <property name="text">
          <string>Export image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_ECG">
   <property name="text">
    <string>&amp;Open ECG</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>CarpetView</class>
   <extends>QGraphicsView</extends>
   <header location="global">carpetview</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
